va/va_prot.h

```diff
@@ -0,0 +1,457 @@
+ */
+
+/**
+ * \file va_prot.h
+ * \brief Protected content API.
+ *
+ * This file contains the \ref api_prot "Protected content API".
+ */
+
+#ifndef VA_PROT_H
+#define VA_PROT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * \defgroup api_prot Protected content API
+ *
+ * @{
+ * \section prolouge Prolouge
+ * Video streaming is ubiquitous and the support for video streaming is widely
+ * available across client open systems such as PCs, MACs, Chromebooks etc. and
+ * closed systems such as setup-box, smart TVs, smart DVDs etc. By default,
+ * video streaming is not considered premium due to various constraints such as
+ * resolution, quality, production cost etc. but recently streaming of premium
+ * video(1080p+) has become norm. The streaming of premium video in open systems
+ * such as PCs, MACs, Chromebooks etc. make video particularly susceptible to
+ * piracies (due to non-video playback usages of such systems) resulting in
+ * millions of dollars loss to content creators.
+ *
+ * Digital Rights Management(DRM) has been proposed to stop piracy of premium
+ * video streams across wide spectrum. There are some known open/closed DRM
+ * standards such as [Widevine by Google](https://www.widevine.com/),
+ * [PlayReady by Microsoft](https://www.microsoft.com/playready/),
+ * [FairPlay by Apple](https://developer.apple.com/streaming/fps/),
+ * [Merlin by Sony](https://www.marlin-community.com/), etc... Each DRM
+ * standard has its properties but all DRM standards support a common
+ * mechanism. This common mechanism involves cryptographical method for
+ * authenticating the client system, delivering bitstream and required
+ * cryptographic assets to client system and then cryptographically processing
+ * bitstream in client system. The cryptographic methods used in these steps
+ * are asymmetric such as RSA, DH etc. and symmetric such as AES CTR, CBC etc.
+ * encryption mechanisms. The authentication of client system, delivery of
+ * bitstream and cryptographic assets to client system is performed using
+ * asymmetric cryptographic mechanism while bitstream is encrypted and processed
+ * using symmetric cryptographic. In DRM world, authentication of client system,
+ * delivery of bitstream and required cryptographic assets to client system is
+ * loosely called provisioning and license acquisition while the processing of
+ * cryptographically secure bitstream is divided as video decryption/decoding,
+ * audio decryption/playback, video display. Besides DRM standards, Video/Audio
+ * bitstream encryption standard such as
+ * [Common Encryption Standard(CENC)](https://www.iso.org/standard/76597.html)
+ * provides a mechanism to normalize bitstream encryption methods across vendors
+ * while providing flexibility.
+ *
+ * \section DRM Pipeline
+ * The most of DRM standards execute following deep pipeline to playback
+ * contents on client systems from streaming serveres - provisioning uses
```

Comment:
s/serveres/servers/ here and everywhere in the document.

Reply:
Corrected.